desktop-widgets/plugins/facebook/facebookconnectwidget.cpp

```diff
@@ -205,13 +205,13 @@ void FacebookManager::sendDive()
 	request.setRawHeader(QByteArray("Content-Length"), QString::number(data.length()).toLocal8Bit());
 	QNetworkReply *reply = am->post(request,data);
 
-    // Why I'm blocking here?
-    // TODO: Move this so it's unblocking.
-	QEventLoop loop;
-	connect(reply, &QNetworkReply::finished, &loop, &QEventLoop::quit);
-	loop.exec();
```

Comment:
So this removes the complete changes made in the previous commit.
please combine those two when you redo the PR just to avoid the back-and-forth
(this second commit is what we should be doing in the first place, I think)